handbook/product/licensing.md

```diff
@@ -0,0 +1,28 @@
+# Licensing
+
+Licensing at Sourcegraph can be confusing: the word "license" (or "licence", for the non-Americans in the crowd) is used in two overlapping contexts within Sourcegraph:
+
+1. The software license under which our users obtain and potentially modify our source code, and
+2. The [team or enterprise plan](https://about.sourcegraph.com/pricing) that a customer uses to gain access to enterprise-only features such as [batch changes](https://docs.sourcegraph.com/batch_changes), which is controlled via a [license key](../ce/license_keys.md).
+
+## Software licensing
+
+The core of Sourcegraph is [licensed under the Apache License, version 2.0](https://github.com/sourcegraph/sourcegraph/blob/main/LICENSE.apache).
+
+As described [in the open source FAQ](../../community/faq.md#is-all-of-sourcegraph-open-source), users can use just that functionality without agreeing to any enterprise licensing terms by building their own server image. If they do so, no code from our enterprise licensed features nor any trademarks or logos will be included in their Sourcegraph deployment. This means that there will be no references to functionality such as batch changes.
+
+Most users, however, will use our prebuilt images, which include code licensed under our [enterprise license](https://github.com/sourcegraph/sourcegraph/blob/main/LICENSE.enterprise). Users who use these images — with or without a paid plan — are agreeing to be bound by the terms of the enterprise license in addition to the terms of the Apache License.
```

Comment:
As well as our TOS, Privacy policy, etc., right @tammy-zhu ? Should we add to this or is this sufficient given the context of this section is clearly about the code license?

Reply:
Added links to address this.